assets/training/model_management/src/azureml/model/mgmt/utils/common_utils.py

```diff
@@ -221,3 +223,68 @@ def fetch_huggingface_model_info(model_id) -> ModelInfo:
         raise AzureMLException._with_error(
             AzureMLError.create(HuggingFaceErrorInFetchingModelInfo, model_id=model_id, error=e)
         )
+
+
+def get_system_time_utc():
+    """Return formatted system time in UTC."""
+    return "{0:%Y-%m-%d %H:%M:%S}".format(datetime.utcnow())
+
+
+def get_file_or_folder_size(path: Path, size: int = 0) -> int:
+    """Return file or folder size in bytes."""
+    if os.path.isfile(path):
+        return os.stat(path).st_size
+    for entry in os.scandir(path):
+        size += get_file_or_folder_size(entry)
+    return size
+
+
+def round_size(size: int) -> str:
+    """Round size."""
+    CONST = 1024
+    dim = ["B", "KB", "MB", "GB", "TB"]
+    count = 0
+    while size / CONST > 1:
+        count += 1
+        size /= CONST
+    return f"{size:.2f} {dim[count]}"
+
+
+def get_vm_available_space_in_kb():
+    """Return available VM size in KB.
+
+    :return: Return available size on VM in KB
+    :rtype: int
+    """
+    try:
+        cmd = "df -k | awk '/\/$/ {print $4}'"
```

Comment:
Don't have have `os` package methods for this?

Reply:
Let me check

Reply:
updated